lib/inc/cpp-pcp-client/connector/connection.hpp

```diff
@@ -90,6 +90,9 @@ using CloseCode = CloseCodeValues::value_;
 
 class LIBCPP_PCP_CLIENT_EXPORT Connection {
   public:
+    /// To keep track of WebSocket timings
+    ConnectionTimings timings;
```

Comment:
Since the new AssociationTimings will be stored in the Connector, it makes more sense to me to allow Connector accessing directly the ConnectionTimings instead of having a copy from a Connection getter call. This is done considering that the actual user interface is provided by Connector, that does not expose the Connection.

No strong opinion here, I'm happy to make it private again if you think it's the case.